### PR TITLE
created fruit-shop-tests

### PR DESCRIPTION
**Pull request by kvjohnny**

### CODE REVIEW

File: src/test/java/core/basesyntax/service/impl/FruitServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        int actual = Storage.remainsOfFruits.get(fruitTransaction.getFruit());
+        assertEquals(expected, actual, "Expected quantity of fruit \""
+                + fruitTransaction.getFruit() + "\" is " + expected + ", but "
+                + "actual is " + actual + ".");
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.remainsOfFruits.clear();
+    }
```
Comment (okuzan) [inline src/test/java/core/basesyntax/service/impl/FruitServiceImplTest.java:59]: better put it after beforeeach

File: src/test/java/core/basesyntax/service/impl/WriterImplTest.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    @Test
+    void writeToFile_correctWritingToOutputFile_ok() {
+        writer.writeToFile(CORRECT_PATH_OF_OUTPUT_FILE, CORRECT_DATA_OF_REPORT);
+        String actual;
+        try (Stream<String> fileLines = Files.lines(Paths.get(CORRECT_PATH_OF_OUTPUT_FILE))) {
+            actual = fileLines.collect(Collectors.joining(System.lineSeparator()));
+        } catch (IOException e) {
+            throw new FruitShopException("Can't read data from file by path "
+                    + CORRECT_PATH_OF_OUTPUT_FILE);
+        }
```
Comment (okuzan) [inline src/test/java/core/basesyntax/service/impl/WriterImplTest.java:67]: IMO there's no need for try catch, test will fail anyway if IOException is thrown, just add it to the method signature

File: src/test/java/core/basesyntax/service/impl/WriterImplTest.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class WriterImplTest {
+    private static final String INCORRECT_PATH_OF_OUTPUT_FILE
+            = "src/test/service/resources/output.csv";
```
Comment (okuzan) [inline src/test/java/core/basesyntax/service/impl/WriterImplTest.java:17]:
```suggestion
            = "incorrect/path/to/file/output.csv";
```
Make it immediately obvious

**Review by okuzan — APPROVED** [review]
Good job

**Comment by kvjohnny** [comment]
Thank you so much!

**Pull request closed without merging**